AirBench/Controllers/BenchController.cs

```diff
@@ -15,7 +15,7 @@ namespace AirBench.Controllers
     public class BenchController : BaseController
     {
         [Authorize]
-        public ActionResult Create()
+        public ActionResult Create(decimal? latitude, decimal? longitude)
```

Comment:
Don't you need to set these parameter values on your `CreateBench` model if they're available?